compose/ui/ui/api/desktop/ui.api

```diff
@@ -3130,6 +3130,7 @@ public abstract interface class androidx/compose/ui/platform/PlatformContext {
 	public fun calculateLocalPosition-MK-Hz9U (J)J
```

Comment:
We have an issue on Windows with `LayerType.OnWindow` - popups bounces:

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/a24f7b92-cd57-4adb-877a-ac46374dfd24

Before the PR, it doesn't bounce.

Reply:
It was a combination of https://github.com/JetBrains/compose-multiplatform/issues/4484 and measuring clipped graphics. Should be much better now

Reply:
Because of https://github.com/JetBrains/compose-multiplatform/issues/4475, it still "jumps"

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/cb2232a7-4459-484c-b5fa-83bbc9aa0b99

before:

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/ea1dd4c0-7ae9-405d-9995-34f138c3ccfa

It is visible, because the non-transparent rect became bigger. 

It is less usable in this state, so I suggest proceed this way:
- fix jumps on iOS (do we still have them after this PR?)
- merge this PR
- keep in mind that `OnWindow` isn't usable on desktop without fixing https://github.com/JetBrains/compose-multiplatform/issues/4475
- add a link to https://github.com/JetBrains/compose-multiplatform/issues/4475 and https://github.com/JetBrains/compose-multiplatform/issues/4484 in KDoc of `OnWindow`